I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 50, in script
    call screen tavern_nav
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "game/scripts/eventscreens.rpy", line 29, in execute
    screen tavern_nav:
  File "game/scripts/eventscreens.rpy", line 29, in execute
    screen tavern_nav:
  File "game/scripts/eventscreens.rpy", line 33, in execute
    imagebutton auto "tavern_bin_%s":
Exception: Imagebutton does not have a idle image. (auto='tavern_bin_%s').

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 50, in script
    call screen tavern_nav
  File "C:\_apps\renpy-8.0.3-sdk\renpy\ast.py", line 2259, in execute
    self.call("execute")
  File "C:\_apps\renpy-8.0.3-sdk\renpy\ast.py", line 2241, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\statements.py", line 342, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\exports.py", line 3336, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\core.py", line 3499, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\core.py", line 3969, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\core.py", line 605, in visit_all
    d.visit_all(callback, seen)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\core.py", line 605, in visit_all
    d.visit_all(callback, seen)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\core.py", line 605, in visit_all
    d.visit_all(callback, seen)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\screen.py", line 476, in visit_all
    callback(self)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\core.py", line 3969, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\screen.py", line 487, in per_interact
    self.update()
  File "C:\_apps\renpy-8.0.3-sdk\renpy\display\screen.py", line 680, in update
    self.screen.function(**self.scope)
  File "game/scripts/eventscreens.rpy", line 29, in execute
    screen tavern_nav:
  File "game/scripts/eventscreens.rpy", line 29, in execute
    screen tavern_nav:
  File "game/scripts/eventscreens.rpy", line 33, in execute
    imagebutton auto "tavern_bin_%s":
  File "C:\_apps\renpy-8.0.3-sdk\renpy\ui.py", line 948, in _imagebutton
    idle = choice(idle, idle_image, "idle", required=True)
  File "C:\_apps\renpy-8.0.3-sdk\renpy\ui.py", line 942, in choice
    raise Exception("Imagebutton does not have a %s image. (auto=%r)." % (name, auto))
Exception: Imagebutton does not have a idle image. (auto='tavern_bin_%s').

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.0.23051307
my vn 1.0
Mon May 29 17:27:14 2023
